imaging/imageRendering.ts

```diff
@@ -1035,7 +1035,7 @@ export const rotateImageRight = function (elementId: string | HTMLElement) {
  * @param {Object} defaultProps - Optional default properties
  * @return {Object} data - A data dictionary with parsed tags' values
  */
-const getSeriesData = function (
+export const getSeriesData = function (
```

Comment:
Se la esporti devi definirla anche nel menu in alto, e metterla nella sezione dove ci sono quelle esportate.
In ogni caso la lascerei non esportata (vedi commenti sotto)